pages/home.html

```diff
@@ -35,29 +40,80 @@ <h2>
   agency_image="fec.svg"
   %}
 
-<section class="usa-section grid-container">
-  <section class="grid-gap">
-    <h2>We support your mission</h2>
-    <section class="tablet:grid-col-8">
-      <h3 class="p-line-height p-normal pad-bottom-mobile">
-        As an office within the General Services Administration, we know how to
-        work with government. Federal, state, and local governments can partner
-        with 18F to deliver projects that fulfill their mission, stay within
-        budget, and use leading technology practices.
-      </h3>
-    </section>
-  </section>
+<section class="usa-section bg-base-lightest padding-y-0">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-6 padding-y-2 tablet:padding-y-6">
```

Comment:
An idea to try, we get rid of the `tablet:padding-y-6` here and target the parent `grid-row` (should be specific to this section, so maybe adding a class name?) and add a css rule `align-items: center`

Reply:
I'll give it a shot